ToDoApp/Backend/src/resolvers/todo/todos.spec.js

```diff
@@ -1,35 +1,75 @@
-const { mergeResolvers } = require("merge-graphql-schemas");
-const { createTestClient } = require('apollo-server-testing');
-const { ApolloServer, gql } = require('apollo-server');
-const { typeDefs } = require("../../schema/typeDefs");
-const { userResolver } = require("../user/userResolver");
-const { todoResolver } = require("./todoResolver");
+const {
+    mergeResolvers
+} = require("merge-graphql-schemas");
+const {
+    createTestClient
+} = require('apollo-server-testing');
+const {
+    ApolloServer,
+    gql,
+    makeExecutableSchema
+} = require('apollo-server');
+const {
+    typeDefs
+} = require("../../schema/typeDefs");
+const {
+    userResolver
+} = require("../user/userResolver");
+const {
+    todoResolver
+} = require("./todoResolver");
+const {
+    augmentSchema
+} = require("neo4j-graphql-js");
+const neo4j = require('neo4j-driver');
 
+const driver = neo4j.driver(
+    'bolt://localhost',
+    neo4j.auth.basic('neo4j', 'password'), {
+        disableLosslessIntegers: true
+    }
+)
 const resolvers = mergeResolvers([userResolver, todoResolver]);
 
-const server = new ApolloServer({typeDefs, resolvers});
-const { query, mutate } = createTestClient(server);
+const schema = makeExecutableSchema({
+    typeDefs,
+    resolvers
+});
+const augmentedSchema = augmentSchema(schema);
+
+const server = new ApolloServer({
+    schema: augmentedSchema,
+    context: {
+        driver
+    }
+});
 
+const {
+    query,
+    mutate
+} = createTestClient(server);
+/**
 describe('query', () => {
     describe('todos', () => {
-        it('todo_list has 2 initial todos', async () => {
+        it('querying all todos returns only 1 Todo because of limit', async () => {
```

Comment:
```suggestion
        it('returns only 1 Todo because of limit', async () => {
``` 
"it returns only 1 Todo because of limit" is more like an English sentence than "it querying all todos returns only 1 Todo because of limit"